src/components/ui/SelectionDrawer/SelectionDrawerUsageExample.tsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * This file is used as an example for the Drawer component.
+ * Once you're done with the example, you can delete this file.
+ */
+'use client';
```

Comment:
`SelectionDrawerButton` is going to be used at Group Collection and Group Single page, but I was planning to use Server Component not Client Component for these pages (if possible).
Do you know if we can apply this Drawer component to Server Component?
And I thought either of following actions would be useful for our upcoming implemantation. Could you please check?
- Directly implement sample component using this Drawer at Server Component(Group Collection or Group Single page)
- Prepare Example for Server component

Reply:
> Do you know if we can apply this Drawer component to Server Component?

I think it should work. I understand that using 'use client' does not affect parent components but child components.

> - Prepare Example for Server component

So, I suppose there is nothing to do with examples of usage in server components.

Reply:
I agree. You can always call client component from server component, 
so GroupPage component or other page component can be server component, and still can call this Drawer component as a child

Reply:
> - Directly implement sample component using this Drawer at Server Component(Group Collection or Group Single page)

It's super simple. Import the `SelectionDrawer` component, for example, in the `src/components/page/FoodsPage.tsx`, a server component.

```ts
import { SelectionDrawerWithButtons } from '@/components/ui/SelectionDrawer/SelectionDrawerUsageExample';

export const FoodsPage = () => {
  return (
    <div>
      <SelectionDrawerWithButtons />
    </div>
  );
};
```
I can include the above code in this PR, but it's as simple as we don't even need to add.

Reply:
Does it make sense?

Reply:
Sorry, make sense, thank you for checking!!!